config/index.ts

```diff
@@ -4,11 +4,11 @@ dotenv.config();
 export const config = {
   baseURL: process.env.BASE_URL ?? "https://archive.org",
   patronUser: {
-    email: process.env.B_EMAIL || '',
-    password: process.env.B_PASSWORD || ''
+    email: process.env.PATRON_EMAIL || '',
+    password: process.env.PATRON_PASSWORD || ''
   },
   privUser: {
-    email: process.env.A_EMAIL || '',
-    password: process.env.A_PASSWORD || '',
+    email: process.env.ARCHIVE_EMAIL || '',
```

Comment:
if this is special, then can we call these vars something else to match their special designation?

Reply:
yes... but for now it was just what's agreed upon